Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the Include Granted Scopes to determine if this authorization request should use
+        /// incremental authorization. If this is provided with the value <c>true</c>, and the authorization 

```

Comment:
Please add a link to incremental authorization - https://developers.google.com/+/web/api/rest/oauth#incremental-auth